src/idl_gen_general.cpp

```diff
@@ -203,8 +203,13 @@ class GeneralGenerator : public BaseGenerator {
       const std::string &classcode, bool needs_includes) {
     if (!classcode.length()) return true;
 
-    std::string code;
-    code = code + "// " + FlatBuffersGeneratedWarning();
+    std::string code("//------------------------------------------------------------------------------\n"
```

Comment:
I took the output of xsd.exe as a sample see here.
https://github.com/samadhicsec/fuzzware/blob/4b516b3d2a9be5000baffee4f3efbdaf886e3c04/Schemas/AutoGenerated/Configuration.cs